tests/phpunit/base-schema.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
Is it a base class for external plugins usage? if yes maybe think to make it a trait that will be more generic and not depends on the `Elementor_Test_Base`

Reply:
its uses `mock_wp_api` that are within `Base_Elementor`. All the extras that within the `elementor_test_base`
```
Base_Elementor, Extra_Assertions, Auth_Helpers
```

Are just good for this base.